examples/match_statements/src/main.sw

```diff
@@ -37,6 +37,11 @@ fn main(num: u64) -> u64 {
         Weather::Snowy => 20,
     };
 
+    let isSunny = match currentWeather {
```

Comment:
```suggestion
    let is_sunny = match current_weather {
```

Reply:
I'd rather follow the convention within the file, but maybe we should change the whole file to use snake case for identifiers.

Reply:
my vote is to change all occurrences to snake case.